phpunit.xml.dist

```diff
@@ -1,7 +1,7 @@
 <phpunit bootstrap="vendor/autoload.php" colors="true">
   <testsuites>
     <testsuite name="formatters">
-      <directory prefix="" suffix="Test.php">tests</directory>
+      <directory prefix="" suffix="Test.php">tests/src</directory>
```

Comment:
Don't move the test file into `tests/src`. That directory is for test utility classes that the autoloader loads for phpunit. Phpunit will load and test `tests/*Test.php`.